ark/attest/tsVersioning.ts

```diff
@@ -41,7 +42,7 @@ export const forTypeScriptVersions = (
 			try {
 				if (existsSync(tsPrimaryPath)) unlinkSync(tsPrimaryPath)
 
-				symlinkSync(targetPath, tsPrimaryPath)
+				symlinkSync(targetPath, tsPrimaryPath, "junction")
```

Comment:
This is also from the other PR correct? It's possible though that this does break on Windows as the cross-os CI doesn't run across multiple TS versions.

Reply:
The version that doesn't touch default TS was not on main yet so I did need this

Reply:
Still, I'd rather `pnpm testTsVersions` work on Windows as well, so you can leave this in. Just not sure about the build stuff from the other PR.

Reply:
I'll may drop this commit later